### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by cf-natali** [janestreet/magic-trace#275] [General]
Hi!

Would it be possible to make a new release, in particular to support hosts without Intel PT (e.g. AMD), thanks to the perf sampling mode added in https://github.com/janestreet/magic-trace/pull/224 ?

Cheers,

**Comment by Emilycroxall1996** [comment]
> Hi!
> 
> Would it be possible to make a new release, in particular to support hosts without Intel PT (e.g. AMD), thanks to the perf sampling mode added in https://github.com/janestreet/magic-trace/pull/224 ?
> 
> Cheers,

**Comment by Xyene** [comment]
This is done! See https://github.com/janestreet/magic-trace/releases/tag/v1.2.0
